Projects/Contracts/Sources/Models/ContractModels.swift

```diff
@@ -513,8 +510,8 @@ extension InsuredPeopleConfig {
     ) {
         let store: ContractStore = globalPresentableStoreContainer.get()
         self.init(
-            currentAgreementCoInsured: contract.currentAgreement?.coInsured ?? [],
-            upcomingAgreementCoInsured: contract.upcomingChangedAgreement?.coInsured,
+            currentAgreementCoInsured: contract.coInsured,
+            upcomingAgreementCoInsured: contract.coInsured,
```

Comment:
since currentAgreementCoInsured && upcomingAgreementCoInsured are same, we can have one property here like:
currentCoinsured

This will probably affect logic on some parts, but eventually make it easier to understand and read